applications/crossbar/priv/couchdb/schemas/system_config.services.json

```diff
@@ -24,6 +28,7 @@
         "scan_rate": {
             "default": 20000,
             "description": "services scan rate",
```

Comment:
Can you update the descriptions to include what the units are? Or migrate from `scan_rate` to `scan_rate_ms`?